test/addons-napi/test_new_target/binding.c

```diff
@@ -47,12 +49,8 @@ napi_value OrdinaryFunction(napi_env env, napi_callback_info info) {
   bool result;
   napi_value newTargetArg;
   NAPI_CALL(env, napi_get_new_target(env, info, &newTargetArg));
-  napi_value undefined;
```

Comment:
Is this test change from upstream?

Reply:
This test doesn't exist upstream, @boingoing added it to node-chakracore.  I'm planning to upstream the test as well.  I found the issue when trying to build and test the V8 version of our branch.